build.gradle

```diff
@@ -45,11 +45,11 @@ dependencies {
 
     implementation "net.dv8tion:JDA:5.0.0-alpha.9"
     implementation "com.github.BlitzOffline:Pagination-Utils:master-SNAPSHOT"
-    implementation 'pw.chew:jda-chewtils:1.24.1'
+    implementation "pw.chew:jda-chewtils:1.24.1"
 
     implementation "com.github.Mokulu:discord-oauth2-api:1.0.2"
 
-    implementation "com.sedmelluq:lavaplayer:1.3.78"
+    implementation group: 'com.github.walkyst', name: 'lavaplayer-fork', version: '1.3.97'
```

Comment:
Please use the short version like the other declarations do

Reply:
ok